tests/test_business.py

```diff
@@ -410,3 +425,51 @@ def test_equality(self):
 
         assert boh1 != boh3
         assert hash(boh1) != hash(boh3)
+
+    def test_get_opening_hours_for_day_success_without_timezone(
+        self, business_hours: BusinessOpeningHours
+    ):
+        target_date = date(2023, 7, 17)  # Monday
+        expected_tz = zoneinfo.ZoneInfo(business_hours.time_zone_name)
+        expected_opening_time = datetime(2023, 7, 17, 7, 0, tzinfo=expected_tz)
+        expected_closing_time = datetime(2023, 7, 17, 22, 0, tzinfo=expected_tz)
+
+        opening_hours_for_day = business_hours.get_opening_hours_for_day(target_date)
+
+        assert opening_hours_for_day[0][0] == expected_opening_time
+        assert opening_hours_for_day[0][1] == expected_closing_time
```

Comment:
please also add a negative assertion, i.e. make sure the opening hours of Tuesday are NOT returned